installer.js

```diff
@@ -123,7 +123,7 @@ function driverInstaller(browserName, browserVersion, targetPath)
     else
     {
         throw new Error(
-            `Browser name "${browserName}" is not a valid browser name. Valid browser names are: ${(VALID_BROWSER_NAMES).join(', ')}`
+            `browser name "${browserName}" is not a valid browser name. Valid browser names are: ${(VALID_BROWSER_NAMES).join(', ')}`
```

Comment:
<blah> is not a valid browser name. The valid names are: <blah>

Reply:
fixed